RecoLocalTracker/SiPixelRecHits/python/CustomPixelCPEGeneric.py

```diff
@@ -0,0 +1,29 @@
+'''Customization functions for cmsDriver to change the phase-2 tracker local reconstruction for configuration different from regular planar rectangular pixels'''
+import FWCore.ParameterSet.Config as cms
+
+def esproducers_by_type(process, *types):
```

Comment:
```suggestion
def _esproducers_by_type(process, *types):
```
to avoid exporting.

BTW, there are already two definitions `def esproducers_by_type(process, *types):` in CMSSW

Reply:
OK. I did not want to use the one in 
https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Configuration/python/common.py#L18 as it looked odd to me to make an include of an HLT package in `RecoLocalTracker`

Reply:
the method is simple enough that it seems OK to me to even re-implement it; I'm just proposing not to make it easily exportable

Reply:
I actually already pushed to my branch as you propose.